addon/templates/components/ember-wormhole.hbs

```diff
@@ -0,0 +1,3 @@
+{{ember-wormhole-placeholder registry=registry as="head" dom=dom}}
```

Comment:
Make a helper `ember-wormhole-placeholder` that returns the value of invoking its first param.

Instead of passing all the internals, pass a bound function
```
{{ember-wormhole-placeholder _createHead~}}
{{yield~}}
{{ember-wormhole-placeholder _createTail~}}
```